src/app/api/community/malysis/reports/[analysisId]/route.ts

```diff
@@ -0,0 +1,33 @@
+import { apiErrorHandler } from "@/lib/api/error";
+import { createMalwareAnalysisServiceClient } from "@/lib/rpc/client";
+import { NextApiRequest } from "next";
+import { NextResponse } from "next/server";
+
+async function handleGET(
+  _req: NextApiRequest,
+  { params }: { params: Promise<{ analysisId: string }> },
+) {
+  const analysisId = (await params).analysisId;
+  if (!analysisId) {
+    throw new Error("Analysis ID is required");
+  }
+
+  const communityTenantId = process.env.COMMUNITY_API_TENANT_ID as string;
+  const communityApiKey = process.env.COMMUNITY_API_KEY as string;
+
+  if (!communityTenantId || !communityApiKey) {
+    throw new Error("Community API credentials are required");
+  }
+
+  const client = createMalwareAnalysisServiceClient(
```

Comment:
@kunaldevxxx Here we are creating a client for our Malware Analysis service based on code generated from: https://buf.build/safedep/api/docs/main:safedep.services.malysis.v1